src/pybind/mgr/nfs/utils.py

```diff
@@ -82,3 +86,22 @@ def check_fs(mgr: 'Module', fs_name: str) -> bool:
     '''
     fs_map = mgr.get('fs_map')
     return fs_name in [fs['mdsmap']['fs_name'] for fs in fs_map['filesystems']]
+
+
+def check_cephfs_path(client: CephfsClient, fs: str, path: str):
```

Comment:
OK. This bit has changed from the last time I saw this. This used to live in export.py source. Any reason to move it here? IIRC, the exceptions were correctly handled there and now this routine returns an (errno, str) tuple. What else has changes? @dparmar18

Reply:
Was this done to debug the make check failure? @dparmar18

Reply:
@vshankar fs is not monkey patched in test_nfs.py, therefore in order to make sure we get past this, I created this helper func that basically does the same thing as it was done in export.py. Also this makes it look clean. This now is monkey-patched to `(0, "cephfs path is valid")`

Reply:
this is the same procedure used in test_nfs, i.e. you will find `check_fs` which is patched to `mock.patch('nfs.ganesha_conf.check_fs', return_value=True)`.

The reason why i didnt monkey patch `fs_client` is then I had to mimic a `fs` which is sort of an overkill and thus creating helper like `check_cephfs_path` and returning value that you'd expect it on a successful trial is a better option to go with